### DISCUSSION TITLE
Simplex 4D In Vector Noise

### DISCUSSION THREAD

**Discussion by adingess** [JacquesLucke/animation_nodes#1825] [Q&A]
Hello, I'm trying to create a seamless loop using AN. The problem I have is that using the Vector Wiggle doesn't provide seamless loops. I was hoping to use the following methodology to make this happen, but I need 4D Simplex in the Vector Noise Node to achieve this. Unless there is another approach out there already to achieve the same results? Just trying to find a way to makes seamless loops in AN and I can't seem to find a way. 
Thank you.

Link to  concept (Start at 15:40): 
https://vimeo.com/341917228?embedded=true&source=video_title&owner=4579794

**Comment by OmarEmaraDev** [comment]
Are looking to loop 1D noise or a 2D one?

**Comment by adingess** [comment]
I believe your noise is 3D and I'm looking for 4D.
![Untitled-1](https://user-images.githubusercontent.com/70594778/151394520-65d3edcc-81e2-4ad9-93ea-233dbe1c507f.png)

**Comment by adingess** [comment]
Or if there is another way to make a seamless loop using wiggle or noise? Trying to use animation nodes to change the color of instanced objects with proximity to another object.

**Reply by OmarEmaraDev** [reply]
There could be a way. What I am asking is as follows, without looping, are you evaluating the noise using a single 1D value (The distance/proximity to an object)?

**Comment by adingess** [comment]
TBH I have no clue, I was just following the tutorial in the link and trying to re-create it in Blender. I've found a new workaround using Geo nodes in the meantime, so I guess I don't need this after all. :)
![Instanced Dynamic Materials](https://user-images.githubusercontent.com/70594778/151435614-d95375e4-ffa7-4fc7-9672-409d7551e296.png)
